### ISSUE TITLE
Garments of Darkness

### ISSUE THREAD

**Issue by netherspite** [netherspite/cc-buglist#423]
<a href="https://github.com/ccnim"><img src="https://avatars3.githubusercontent.com/u/10582080?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [ccnim](https://github.com/ccnim)**    
_Tuesday Apr 14, 2015 at 20:37 GMT_  
_Originally opened as https://github.com/ccshiro/cc-buglist/issues/423_

----

This undead priest quest will have you heal and buff a wounded guard. The Dwarf equivalent quest had the dwarf say a few things but for this quest I couldn't find anything.

Basically what I need is the guard's says and if possible a video show-casing the quest.

**Comment by netherspite** [comment]
<a href="https://github.com/nz54445"><img src="https://avatars1.githubusercontent.com/u/8640523?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [nz54445](https://github.com/nz54445)**  
_Wednesday Apr 15, 2015 at 10:34 GMT_

----

The alliance side of the quest chain is called In Favor of light and Garments of light, the wowwiki entry is here: http://www.wowwiki.com/Quest:In_Favor_of_the_Light_%28Dun_Morogh%29

Now i could not find anything on the dwarf quest, but i did find one for humans (which is the identical quest chain with the identical reward): http://www.wowwiki.com/Quest:In_Favor_of_the_Light_%28Elwynn_Forest%29

I also found a video demonstrating the human quest chain here: https://www.youtube.com/watch?v=6tKPJzl37ug

It would appear that the wowwiki has it down word for word for the humans and the descriptions of the human quest, so i would think that it is a good enough source for the dwarf quest as well.

The Garments of light quests on wowwiki are listed here: http://www.wowwiki.com/Quest:Garments_of_the_Light_%28Elwynn_Forest%29 and http://www.wowwiki.com/Quest:Garments_of_the_Light_%28Dun_Morogh%29

I would not think that the mountaineer Dolf (the object of the quest for dwarves) would say anything else besides the generic line that the human also has. Hopefully this helps.

**Comment by netherspite** [comment]
<a href="https://github.com/ccnim"><img src="https://avatars3.githubusercontent.com/u/10582080?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ccnim](https://github.com/ccnim)**  
_Wednesday Apr 15, 2015 at 17:24 GMT_

----

Seems like my post wasn't all that clear.

I've already completed the dwarf quest line when I was doing Dun Morogh but when I was working on Tirisfal Glades I couldn't find any information about the counterpart for undead priests, so basically I need help researching the undead priest class quest Garments of Darkness.

**Comment by netherspite** [comment]
<a href="https://github.com/nz54445"><img src="https://avatars1.githubusercontent.com/u/8640523?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [nz54445](https://github.com/nz54445)**  
_Wednesday Apr 15, 2015 at 22:26 GMT_

----

My mistake i thought you needed the dwarven part. I'll try to look into the undead part, all I have so far is a video that is taken in cata or some other that is so poor it shows nothing.

**Comment by netherspite** [comment]
<a href="https://github.com/nz54445"><img src="https://avatars1.githubusercontent.com/u/8640523?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [nz54445](https://github.com/nz54445)**  
_Thursday Apr 16, 2015 at 07:49 GMT_

----

@ccnim Just to be sure that i understand it, you need what the deathguard Kel is saying right? I can't find a video anywhere of him having a speech bubble, nothing on wowhead, wowwiki etc. It's possible that his line is exactly the same as the alliance line since it seems quite generic of a line: Ah, %Class you came along just in time. I appreciate it.

**Comment by netherspite** [comment]
<a href="https://github.com/ccnim"><img src="https://avatars3.githubusercontent.com/u/10582080?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ccnim](https://github.com/ccnim)**  
_Friday Apr 17, 2015 at 15:34 GMT_

----

Hmm, I'll leave him silent until we find something solid.

**Comment by netherspite** [comment]
<a href="https://github.com/Wtfeater"><img src="https://avatars0.githubusercontent.com/u/10584393?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [Wtfeater](https://github.com/Wtfeater)**  
_Friday Apr 17, 2015 at 19:12 GMT_

----

Although probably not rock solid proof, I found this list from trinitycore showing the different wording for the races, maybe googleing parts of the sentences will yield some results? https://gitlab.com/trinitycore/trinitycore/raw/7252f11add985247d7ae5b8262d8f38e60bd8e33/sql/old/3.3.5a/TDB49_to_TDB50_updates/world/2012_12_07_05_world_creature_text.sql

This video confirms atleast the gnome/dwarf one to be correct, as you've stated.
https://www.youtube.com/watch?v=ptXkPzIJCGI&feature=youtu.be&t=124

**Comment by netherspite** [comment]
<a href="https://github.com/nz54445"><img src="https://avatars1.githubusercontent.com/u/8640523?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [nz54445](https://github.com/nz54445)**  
_Friday Apr 17, 2015 at 19:42 GMT_

----

@Wtfeater nice find! I thought as much, the lines seem to be fairly generic. Really damn hard to find an undead video/log, if there is one even. Hopefully this helps nim along.

**Comment by netherspite** [comment]
<a href="https://github.com/ccnim"><img src="https://avatars3.githubusercontent.com/u/10582080?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ccnim](https://github.com/ccnim)**  
_Thursday May 21, 2015 at 13:59 GMT_

----

@Wtfeater I did find these as well but I'm not sure if they made it up or not. Looking at the say 'Farewell to you, and may shadow always protect you!', I'm inclined to say it's made up.

**Comment by netherspite** [comment]
<a href="https://github.com/vove9000"><img src="https://avatars1.githubusercontent.com/u/12601771?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [vove9000](https://github.com/vove9000)**  
_Saturday May 30, 2015 at 03:33 GMT_

----

There is a youtube movie of a guy doing exactly that quest with undead priest: https://youtu.be/J3yWVW_vqoQ

The resolution is so low though, that you can hardly guess what it says. If anyone wants to try his deducting skills (seems to be needed more than just reading capabilities) check it out. After getting the fortitude buff I am almost sure the guard says: "Objective accomplished", or something similar.

I also have no idea whether this is a private realm or retail WoW.

**Comment by netherspite** [comment]
<a href="https://github.com/nz54445"><img src="https://avatars1.githubusercontent.com/u/8640523?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [nz54445](https://github.com/nz54445)**  
_Saturday May 30, 2015 at 08:15 GMT_

----

@vove9000 I saw that video as well. I believe that is another private server, due to the upload date (2012) clashing with the release of cataclysm (2010) when the quest was removed.

**Comment by netherspite** [comment]
<a href="https://github.com/Wtfeater"><img src="https://avatars0.githubusercontent.com/u/10584393?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [Wtfeater](https://github.com/Wtfeater)**  
_Saturday May 30, 2015 at 17:29 GMT_

----

Vove9000's video is indeed recorded on a private server, as can be verified in the chatbox of some other videos the same person uploaded around the same time.

**Comment by netherspite** [comment]
<a href="https://github.com/vove9000"><img src="https://avatars1.githubusercontent.com/u/12601771?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [vove9000](https://github.com/vove9000)**  
_Saturday May 30, 2015 at 20:33 GMT_

----

Then maybe just improvise? Won't be blizzlike, but I also don't think it's much of a gamebreaker if he doesn't speak exactly what he should?
